packages/appstore/src/components/cfds-listing/cfds-listing.scss

```diff
@@ -865,160 +869,6 @@
     }
 }
 
-.account-poa {
```

Comment:
Are you removing everything related to account-poa?

Reply:
for cfd package - yes. we have mentioned it